### DISCUSSION TITLE
pipewire-pulse no longer in repository

### DISCUSSION THREAD

**Discussion by t-braswell** [void-linux/void-packages#50510] [Q&A] [answered]
Im trying to set up pipewire to work, as per instructions from the docs. in order to get sound to work for firefox, i need the pipewire-pulse package (as the docs mention) pulseaudio; however, it doesnt seem to be in the repo anymore (it was there about 2 weeks ago when I tried to set up another machine). Was this on purpose? if it was, how can I enable pipewire to handle pulseaudio applications?

**Comment by classabbyamp** [comment, marked as answer]
the docs do not mention a pipewire-pulse package, it is a program within the pipewire package

**Reply by t-braswell** [reply]
thank you for the response- thought that might be the case and figured that out on my own. just got sound working!

**Reply by od4knb** [reply]
Hi @classabbyamp,

In the [Fedora](https://packages.fedoraproject.org/pkgs/pipewire/pipewire-pulseaudio/) repo the "pipewire-pulse" deamon is called "pipewire-pulseaudio", and it is stored in a "sub package" of the ["pipewire" package](https://koji.fedoraproject.org/koji/packageinfo?packageID=24555).

But, on Debian and Ubuntu, "pipewire-pulse" is a *separate* .deb package. Please, check the [Debian](https://packages.debian.org/search?keywords=pipewire-pulse&searchon=names&suite=all&section=all) and [Ubuntu](https://packages.ubuntu.com/search?keywords=pipewire-pulse&searchon=names&suite=all&section=all) repos.

WKR, Odie
